static/sass/_pattern_card.scss

```diff
@@ -21,94 +17,53 @@
     }
   }
 
-  %post-card-header {
-    border-radius: 2px;
-    padding: $sp-small $sp-medium;
-  }
-
-  .p-card--post {
-    @extend .p-card--highlighted;
-    display: flex !important;
+  %flex-column {
+    display: flex;
     flex-direction: column;
-    padding: 0;
-
-    > .p-card__content {
-      border-top: 1px dotted $color-mid-light;
-      display: flex;
-      flex-direction: column;
-      height: 100%;
-      margin: 0 $sp-x-small;
-      padding: $sp-x-small;
-
-      > a, h3, p {
-        display: block;
-        margin: $sp-x-small 0;
-      }
-    }
-
-    .p-card__footer {
-      border-top: 1px dotted $color-mid-light;
-      font-size: .875rem;
-      margin: auto $sp-x-small 0;
-      max-width: inherit;
-      padding: $sp-medium $sp-x-small;
-    }
-  }
-
-  .p-card--muted {
-    @extend .p-card;
-    background-color: $color-light;
-    margin-bottom: 0;
-    padding: 0;
-    box-shadow: 0 1px 2px 0 transparentize($color-dark, .8);
-
-    > .p-card__header  {
-      margin-bottom: 0;
-      border-top: 3px solid $color-light;
-      border-bottom: 0;
-    }
-
-    > .p-card__content {
-      margin: 0 $sp-x-small $sp-x-small;
-      padding: $sp-medium $sp-x-small $sp-x-small;
-      border-top: 1px dotted $color-mid-light;
-    }
   }
 
-  .p-card__header {
-    @extend %post-card-header;
-
+  .p-card--post {
     &--cloud-and-server {
-      @extend %post-card-header;
-      border-top: 3px solid #a87ca0;
+      @extend %p-card--highlighted;
+      @extend %flex-column;
+      @include vf-m-category-bar(#a87ca0);
```

Comment:
@lyubomir-popov Where is `vf-m-category-bar` defined? Is this going to be part of beta-2?